source/config.js

```diff
@@ -4,11 +4,7 @@
  * Configuration
  */
 
-var secure = window.location.protocol === 'https:';
-
 module.exports = {
     name: 'likely',
-    prefix: 'likely__',
-    secure: secure,
-    protocol: secure ? 'https:' : 'http:',
+    prefix: 'likely__'
```

Comment:
Yep, seems it's because `--fix`. I think we need two different npm scripts: `codestyle-check` and `codestyle-fix`, and run within CI the first one.

I'll fix my code here.